### ISSUE TITLE
Failed to render Hl. TypeError: e.includes is not a function

### ISSUE THREAD

**Issue by lucassusanto** [getinsomnia/insomnia#6868]
### Expected Behavior

Insomnia can render the UI

### Actual Behavior

UI won't render

### Reproduction Steps

1. Open Insomnia

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

```
TypeError: e.includes is not a function
    at qVe (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:470:868)
    at _l (file:///snap/insomnia/239/resources/app.asar/debug-1d1cf6bf.js:53:34237)
    at FS (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:42:19553)
    at oF (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:44:44101)
    at rF (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:44:39819)
    at Gpe (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:44:39745)
    at Bd (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:44:39596)
    at Px (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:44:35959)
    at eF (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:44:34906)
    at N (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///snap/insomnia/239/resources/app.asar/index-12757498.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)
```

### Insomnia Version

8.4.4

### What operating system are you using?

Other (specify below)

### Operating System Version

Pop!_OS 22.04 LTS

### Installation method

Pop!_Shop

**Comment by eveevans** [comment]
Same here. :/

**Comment by IlliaHreben** [comment]
the same

**Comment by aadil-sharif** [comment]
Also experiencing the same after update.

**Comment by ellidani** [comment]
Same issue here after the update.

**Comment by zinyosrim** [comment]
what a pain

**Comment by JSeam2** [comment]
Same here, I'm considering just ditching Insomnia at the rate the library is progressing. Tbh it's better if no updates were even done

**Comment by filfreire** [comment]
Hey folks, we will launch a new release with a fix for this today, 8.4.5.

Closing this as a duplicate of #6857

**Comment by filfreire** [comment]
Update [here](https://github.com/Kong/insomnia/issues/6857#issuecomment-1824752852)
